polymorphic_value.h

```diff
@@ -73,7 +73,7 @@ template<typename T, polymorphic_value_options Options = polymorphic_value_optio
     polymorphic_value() {}
     polymorphic_value(nullopt_t) {}
     polymorphic_value(const polymorphic_value& src) requires copyable {
-        src.m_handler.copy(*this, src.m_data);
+        src.getHandler().copy(*this, src.m_data);
```

Comment:
I found about 10 places where the same pattern recurs. This is all places m_handler is mentioned except for where it is placement newed, basically. Did you figure out that it should not be needed to call getHandler elsewhere?

Reply:
I've updated all locations but still see failures in the tests.  I've converted this PR to a draft until I have time to work through the issues.